tests/test_classifier/test_prcurve.py

```diff
@@ -29,7 +30,6 @@
 from sklearn.datasets import make_regression
 from sklearn.tree import DecisionTreeClassifier
 from sklearn.linear_model import RidgeClassifier
-from sklearn.model_selection import train_test_split as tts
```

Comment:
This might have to also be added back to get the tests to pass ... sorry!

Reply:
Or maybe not ... looks like the tests did pass; I think that's probably because of an `import *` somewhere.